### PR TITLE
Tagalog Localization Updates

### PR DESCRIPTION
**Pull request by jcchikikomori**
## Disclaimer

For everyone especially Filipino folks like me, this is not a 1:1 translation. This PR is based on two criterias that i set:

- Existing Localization/Translation pattern from Nokia Feature phones (back in 2000s or any S30/S40 devices)
- Some translation needed to be adjusted based on modern times, slangs, etc.

## Changes

- Update localization, but requires testing almost everything.
- Some Tagalog localization adjustments on Settings (requires unit testing on my end)

### Progress

- [x] Achievements
- [x] Audio
- [x] Controls
- [x] CwCheats
- [x] DesktopUI
- [x] Developer
- [x] Dialog
- [x] Error
- [x] Game
- [x] Graphics
- [x] InstallZip
- [x] KeyMapping
- [x] MainMenu
- [x] MainSettings
- [x] MappableControls
- [x] MemStick
- [x] Networking
- [x] PSPSettings
- [x] Pause
- [x] PostShaders
- [x] PSPCredits
- [x] RemoteISO
- [x] Reporting
- [x] Savedata
- [x] Screen
- [x] Search
- [x] Store
- [x] SysInfo
- [x] System
- [x] TextureShaders
- [x] Themes
- [x] UI Elements
- [x] Upgrade
- [x] VR

## Screenshots

image

image

image

image

### CODE REVIEW

File: assets/lang/tg_PH.ini [hunk 1]
```diff
@@ -157,125 +157,125 @@ Swipe sensitivity = Swipe sensitivity
...
 Y = Y
 
 [CwCheats]
 Cheats = Mga Daya
-Edit Cheat File = Edit Cheat File
-Import Cheats = Import from cheat.db
-Import from %s = Import ang mga daya mula sa %s
-Refresh interval = Refresh interval
+Edit Cheat File = I-edit ang Cheat File
+Import Cheats = Mag Import sa cheat.db
```
Comment (jcchikikomori) [inline assets/lang/tg_PH.ini:176]: Wrong Grammar. Change it into `I-import ang mga daya mula sa cheat.db`

File: assets/lang/tg_PH.ini [hunk 2]
```diff
@@ -157,125 +157,125 @@ Swipe sensitivity = Swipe sensitivity
...
 
 [DesktopUI]
 # If your language does not show well with the default font, you can use Font to specify a different one.
 # Just add it to your language's ini file and uncomment it (remove the # by Font).
 #Font = Trebuchet MS
 About PPSSPP... = Tungkol sa PPSSPP...
 Auto = Awto
-Auto Max Quality = Auto Max &Quality
-Backend = Rendering &Backend (Restarts PPSSPP)
+Auto Max Quality = Awtomatikong Max ang Kalidad
```
Comment (jcchikikomori) [inline assets/lang/tg_PH.ini:186]: Translation not final on this one

File: assets/lang/tg_PH.ini [hunk 3]
```diff
@@ -157,125 +157,125 @@ Swipe sensitivity = Swipe sensitivity
...
 Enable Cheats = Paganahin ang daya
 Enable Sound = Paganahin ang tunog
 Exit = Lumabas
 Extract File... = Extract File...
 File = File
-Frame Skipping = Frame skipping
-Frame Skipping Type = Frame skipping type
-Fullscreen = Fullscreen
-Game Settings = Settings ng Laro
+Frame Skipping = Pag-skip ng frame
```
Comment (jcchikikomori) [inline assets/lang/tg_PH.ini:209]: Change this to `Pag-talon ng frame`

File: assets/lang/tg_PH.ini [hunk 4]
```diff
@@ -157,125 +157,125 @@ Swipe sensitivity = Swipe sensitivity
...
+Frame Skipping Type = Uri ng pag-skip ng frame
```
Comment (jcchikikomori) [inline assets/lang/tg_PH.ini:210]: Change this to `Uri ng pag-talon ng frame`

File: assets/lang/tg_PH.ini [hunk 5]
```diff
@@ -157,125 +157,125 @@ Swipe sensitivity = Swipe sensitivity
...
+Load .sym File... = I-Load ang .sym File...
+Load Map File... = I-Load ang Map File...
+Load State = I-Load ang State
+Load State File... = I-Load ang State File...
 Log Console = Log Console
 Memory View... = Memory View...
 More Settings... = Karagdagang settings...
-Nearest = Nearest
+Nearest = Pinakamalapit
 Recent = &Recent
```
Comment (jcchikikomori) [inline assets/lang/tg_PH.ini:235]: Translate this to: `Kamakailan`

**Comment by hrydgard** [comment]
Hm, you reviewing your own PR? :)

I'm fine with merging this as it is, but I don't know tagalog, so...

**Comment by jcchikikomori** [comment]
> Hm, you reviewing your own PR? :)
> 
> I'm fine with merging this as it is, but I don't know tagalog, so...

Oh, yeah. I do this mostly on my work/code. Sorry for that!
But i am almost done on this.

**Comment by hrydgard** [comment]
@jcchikikomori hey, you want to get this in for 1.17.1 ?  It will go out later this week.

**Comment by jcchikikomori** [comment]
> @jcchikikomori hey, you want to get this in for 1.17.1 ? It will go out later this week.

I don't think i can due to my busy schedule. But my goal is to finish these next week.

**Pull request merged**